model/tensorflow_hub/dffml_model_tensorflow_hub/tfhub_models.py

```diff
@@ -0,0 +1,141 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2019 Intel Corporation
+"""
+Description of what this model does
+"""
+
+import abc
+import inspect
+from typing import AsyncIterator, Tuple, Any, List, Optional, NamedTuple, Type
+import os
+from dffml.repo import Repo
+from dffml.source.source import Sources
+from dffml.feature import Features
+from dffml.accuracy import Accuracy
+from dffml.model.model import ModelConfig, ModelContext, Model
+from dffml.util.entrypoint import entrypoint
+from dffml.base import BaseConfig, config, field
+from dffml.feature.feature import Feature, Features
+from dffml_model_tensorflow.dnnc import TensorflowModelContext
```

Comment:
I missed the `dffml_model_tensorflow` model import. Just saw this because of the CI.

- Let's do the `tf v1 -> v2` changes for `dffml_model_tensorflow` in another PR
- Once those changes are merged, we'll bump the version to release the v2 compatable one.
- Then we'll add `dffml-model-tensorflow` to the `setup.py` `install_requires` for this package
- CI shouldn't say that it can't import it anymore.

> If we don't do it in another PR we'll have to mess with the CI scripts to only install certain plugins for certain other plugins in development mode. Which we could do and should do eventually, but let's focus on this first.

Reply:
Turns out we don't need dffml-model-tensorflow here, anyway we are already migrating to TF2 in different PR.